packages/block-editor/src/store/reducer.js

```diff
@@ -350,15 +350,18 @@ const withBlockCache = ( reducer ) => ( state = {}, action ) => {
  */
 function withPersistentBlockChange( reducer ) {
 	let lastAction;
+	let markNextChangeAsNotPersistent = false;
```

Comment:
Should we name this with the "is" prefix? Something like isNextActionExplicitlyNotPersistent?

Reply:
That is harder to read.